src/coffee/providers/map-loader.coffee

```diff
@@ -34,6 +38,25 @@ angular.module('uiGmapgoogle-maps.providers')
       isGoogleMapsLoaded = ->
         angular.isDefined(window.google) and angular.isDefined(window.google.maps)
 
+      onWindowLoad = (options)->
+        # if its a WebView
+        if !(!window.cordova && !window.PhoneGap && !window.phonegap && !window.forge)
+          document.addEventListener 'deviceready', ->
+            # Cordova specific https://github.com/apache/cordova-plugin-network-information/
+            if window.navigator.connection && window.Connection && window.navigator.connection.type == window.Connection.NONE
```

Comment:
That checks was included by another PR too. The intent for that checks was to test if the device has no network connection and then wait for the online event, but now if we will check for webview platforms and waiting for 'deviceready' its unnecessary to check for  obj.navigator.connection and obj.Connection. So,  we could just check for connection.type == window.Connection.NONE in order to add the listener for online event.

´´´javascript
onWindowLoad = (options)->
        if isWebView()
          document.addEventListener 'deviceready', ->
            window.navigator.connection.type == window.Connection.NONE
              document.addEventListener 'online', ->
                  if !lastNetworkStatus || lastNetworkStatus != 'online'
                    lastNetworkStatus = 'online';
                    includeScript options if !isGoogleMapsLoaded()
´´´